scripts/template_setup_posix

```diff
@@ -301,6 +307,14 @@ if [ "${do_cmake_pkg}" = "y" ]; then
   echo
 fi
 
+# Create compatibility links for old Zephyr versions
+if [ "${do_old_zephyr}" = "y" ]; then
+  echo "Creating links for old Zephyr compatibility ..."
+  ln -sr gnu/* . || assert_rc "ERROR: Creating toolchain links" 50
+  ln -sr cmake/zephyr/gnu/* cmake/zephyr || assert_rc "ERROR: Creating cmake links" 50
```

Comment:
We also need to symlink `hosttools/*` to `.` for host tools.

Reply:
Thanks. done.